index.js

```diff
@@ -144,6 +145,31 @@ sassLint.lintText = function (file, options, configPath) {
 };
 
 /**
+ * Handles ignored files for plugins such as the gulp plugin. Checks every file passed to it against
+ * the ignores as specified in our users config or passed in options.
+ *
+ * @param {object} file - The file/text to be linted
```

Comment:
The rest of the file is using lower case o's so I wont update this one to be different.